backend/src/main/java/fk/prof/backend/util/proto/RecorderProtoUtil.java

```diff
@@ -0,0 +1,73 @@
+package fk.prof.backend.util.proto;
+
+import fk.prof.backend.proto.BackendDTO;
+import recording.Recorder;
+
+public class RecorderProtoUtil {
+
+  public static Recorder.ProcessGroup mapRecorderInfoToProcessGroup(Recorder.RecorderInfo recorderInfo) {
```

Comment:
As we discussed, proto builders are single-time use only and cannot be used to construct other proto objects once `build()` is called on them